### PR TITLE
Azure Support management SDK for public preview version "2019-05-01-preview"

### PR DESCRIPTION
**Pull request by shahbj79**
This change generates Azure Support management SDK against API version 2019-05-01-preview.
Tests have been added for all the APIs.

### CODE REVIEW

File: sdk/support/Microsoft.Azure.Management.Support/src/AzSdk.RP.props [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (markcowl) [inline sdk/support/Microsoft.Azure.Management.Support/src/AzSdk.RP.props]: This should be unnecessary, given the same file in the parent directory

File: sdk/support/Microsoft.Azure.Management.Support/src/Microsoft.Azure.Management.Support.csproj [hunk 2]
```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.Support</PackageId>
+    <Description>Provides developers with libraries for the creation and management of SupportTicket resource.</Description>
```
Comment (markcowl) [inline sdk/support/Microsoft.Azure.Management.Support/src/Microsoft.Azure.Management.Support.csproj]: This is the description that will appear in Nuget - you should include a couple of sentences about the RP (allows users to manage Azure Support tickets).

**Review by markcowl — DISMISSED** [review]

File: sdk/support/Microsoft.Azure.Management.Support/src/AzSdk.RP.props (hunk 1 above)
Comment (shahbj79) [reply sdk/support/Microsoft.Azure.Management.Support/src/AzSdk.RP.props]: Removed.

File: sdk/support/Microsoft.Azure.Management.Support/src/Microsoft.Azure.Management.Support.csproj (hunk 2 above)
Comment (shahbj79) [reply sdk/support/Microsoft.Azure.Management.Support/src/Microsoft.Azure.Management.Support.csproj]: Updated description.

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**